website/content/docs/enterprise/namespaces/namespace-limits.mdx

```diff
@@ -0,0 +1,110 @@
+---
+layout: docs
+page_title: Running Vault Enterprise with many namespaces
+description: >-
+  Guidance for using thousands of namespaces with Vault Enterprise
+---
+
+# Many namespaces
+
+Namespaces can be used to create isolated environments within Vault Enterprise.
+Vault has a default limit to the number of namespaces that can be created, but
+the limit can be modified if needed. This page provides guidance of what to
+expect when operating a Vault cluster with thousands of namespaces.
+
+# Default limit
+
+@include 'namespace-limits.mdx'
+
+# Modifying the limit
+
+@include 'storage-entry-size.mdx'
+
+# Performance
+
+Running Vault with thousands of namespaces can have operational impacts on a
+cluster. Below are some performance considerations to take into account before
+using thousands of namespaces.
+
+It is **not** recommended to use thousands of namespaces with any version of Vault
+lower than 1.13.9, 1.14.5, or 1.15.0. Improvements were released in those
+versions which can improve the reliability of Raft heartbeats when using many
+namespaces.
+
+All the data below was tested by creating a varying number of namespaces (using
+only the default mounts) on a 3-node Vault cluster using integrated storage. The
+cluster ran on Google Kubernetes Engine. These scenarios were tested using
+n2-standard-16 and n2-standard-32 nodes.
+
+## Unseal time
+
+Immediately after an unseal Vault will set up and initialize every mount, and
+Vault **may be unresponsive** while this initialization is happening. Each
+namespace has at least the 4 default mounts (`sys`, `identity`, `cubbyhole`, and
+`token`) that need to be set up.
+
+Testing a varying number of namespaces, the following post-unseal times
+were observed:
+
+| Number of namespaces | Post-unseal time  |
```

Comment:
Do we want to define what a "Post-unseal" time is?

Reply:
Added